tests/test_cif_io.py

```diff
@@ -177,4 +177,21 @@ def mockreturn(*args, **kwargs):
         assert actual == expected
         # assert True
 
-# End of file.
+
+mpcontribs_tst = [
+    ('mpc', 'This function is not yet implemented'),
+    ('fs', None),
+    (False, 'mpc_client must be either mpc or fs')
+]
+
+
+@pytest.mark.parametrize("inpt, expected", mpcontribs_tst)
+def test_set_client(inpt, expected):
+    result = set_client(inpt)
+
+    # TODO: Implement test cases for unexpected input
```

Comment:
These bad inputs won't change the test in this case.  Rather they will change the functionality in the function.  You don't put logic in the test or you break the entire logic because the test is passing because of the logic in the test, but it should pass because of hte logic in the function itself!

There is one exception which is if you want your function to raise an exception in a certain situation.  In that case, you have to write a new test function `test_set_client_bad()` that checks that the exception is correctly raised when it is supposed to, but this won't be needed here.